nautobot/core/graphql/schema.py

```diff
@@ -159,6 +167,34 @@ def extend_schema_type_null_field_choice(schema_type, model):
     return schema_type
 
 
+def extend_schema_type_filter(schema_type, model):
+    """Extend schema_type object to be able to filter on multiple levels of a query
+
+    Args:
+        schema_type (DjangoObjectType): GraphQL Object type for a given model
+        model (Model): Django model
+
+    Returns:
+        schema_type (DjangoObjectType)
+    """
+    for field_name in dir(model):
+        attr = getattr(model, field_name)
+        # Check attribute is a ManyToOne field
+        if not isinstance(attr, ReverseManyToOneDescriptor):
+            continue
+
```

Comment:
Are we missing a call to `str_to_var_name(field.name)` here, as present in most of the other `extend_` functions?